### ISSUE TITLE
`cyctl` install Cyclops command

### ISSUE THREAD

**Issue by KaradzaJuraj** [cyclops-ui/cyclops#325]
## Feature Request
`cyctl` should be able to install Cyclops into your cluster.

## Solution
Implement a command that installs the latest Cyclops version into your cluster (with the CRDs and default templates) → `cyctl init`
It should have the same functionality as this command:
```bash
kubectl apply -f https://raw.githubusercontent.com/cyclops-ui/cyclops/v0.6.2/install/cyclops-install.yaml && 
kubectl apply -f https://raw.githubusercontent.com/cyclops-ui/cyclops/v0.6.2/install/demo-templates.yaml
```

**Comment by thebigbone** [comment]
Hey, I am working on this. I was thinking if a subcommand for specifying the version would be a good idea? (`latest` would be the default)

**Comment by petar-cvit** [comment]
yeah, you can fallback to `main` if no version provided. Will assign you to the issue
